backend/routes/utils/responses.go

```diff
@@ -4,20 +4,33 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"strings"
 
 	"github.com/gorilla/websocket"
+	"github.com/keep-starknet-strange/art-peace/backend/config"
 	"github.com/keep-starknet-strange/art-peace/backend/core"
 )
 
+func getHttpConfig() config.HttpConfig {
+	return core.ArtPeaceBackend.BackendConfig.HttpConfigValues
+}
+
 func SetupAccessHeaders(w http.ResponseWriter) {
-	w.Header().Set("Access-Control-Allow-Origin", "*")
-	w.Header().Set("Access-Control-Allow-Methods", "GET, POST, PUT, DELETE")
-	w.Header().Set("Access-Control-Allow-Headers", "Content-Type")
+	config := getHttpConfig()
+
+	for _, origin := range config.AllowOrigin {
+		w.Header().Add("Access-Control-Allow-Origin", origin)
```

Comment:
I'm not sure this would work with multiple different origins, I'd recommend either :

1. Use only the one origin for now, and create a TODO to process multiple different origins
2. Follow this js code and create a similar setup here : https://stackoverflow.com/a/54366749

Reply:
This means I have to add a list of allowed origins in the HttpConfig?. Something like:

```
	Http: HttpConfig{
		AllowOrigin:  []string{"https://local.com", "https://home.com"}
        }
```
        
Next, I do this in **utils/responses.go** to check if the request's origin is in the allowed list:


```
func SetupAccessHeaders(w http.ResponseWriter, r *http.Request) {
	config := getHttpConfig()
	origin := r.Header.Get("Origin")
	for _, allowedOrigin := range config.AllowOrigin {
		if allowedOrigin == origin {
			w.Header().Set("Access-Control-Allow-Origin", origin)
			break
		}
	}
```

Reply:
The second option: 

	config := getHttpConfig()

	// TODO: Process multiple origins in the future.
	if len(config.AllowOrigin) > 0 {
		w.Header().Set("Access-Control-Allow-Origin", config.AllowOrigin[0])
	}
	
Then:

        Http: config.HttpConfig{
            AllowOrigin:  []string{"https://domain1.example"}
        },